auth-api/pom.xml

```diff
@@ -10,15 +10,13 @@
   <groupId>io.stargate.auth.api</groupId>
   <artifactId>auth-api</artifactId>
   <properties>
-    <jettyVersion>9.4.24.v20191120</jettyVersion>
     <jerseyVersion>2.30.1</jerseyVersion>
     <jacksonVersion>2.10.1</jacksonVersion>
   </properties>
   <dependencies>
     <dependency>
       <groupId>io.stargate.auth</groupId>
       <artifactId>authentication</artifactId>
-      <version>0.0.15-SNAPSHOT</version>
```

Comment:
Couldn't we just use `${project.version}`?

Reply:
Right that would work too. Either one works for me, I can make the change.

Reply:
Done in e3907ba